pipeline_dp/budget_accounting.py

```diff
@@ -272,18 +273,24 @@ def _compose_distributions(
         composed, pld = None, None
 
         for mechanism_spec_internal in self._mechanisms:
-            if mechanism_spec_internal.mechanism_spec.noise_kind == NoiseKind.LAPLACE:
+            if mechanism_spec_internal.mechanism_spec.noise_kind == MechanismType.LAPLACE:
                 # The Laplace distribution parameter = std/sqrt(2).
                 pld = pldlib.PrivacyLossDistribution.from_laplace_mechanism(
                     mechanism_spec_internal.sensitivity *
                     noise_standard_deviation / math.sqrt(2) /
                     mechanism_spec_internal.weight,
                     value_discretization_interval=self._pld_discretization)
-            elif mechanism_spec_internal.mechanism_spec.noise_kind == NoiseKind.GAUSSIAN:
+            elif mechanism_spec_internal.mechanism_spec.noise_kind == MechanismType.GAUSSIAN:
                 pld = pldlib.PrivacyLossDistribution.from_gaussian_mechanism(
                     mechanism_spec_internal.sensitivity *
                     noise_standard_deviation / mechanism_spec_internal.weight,
                     value_discretization_interval=self._pld_discretization)
+            elif mechanism_spec_internal.mechanism_spec.noise_kind == MechanismType.GENERIC:
+                epsilon_0 = math.sqrt(2) / noise_standard_deviation
```

Comment:
Please add comment like this before this line (please feel free to update this suggested comment)

\#  The calibration is performed by one parameter. So it is required to make some conversion between noise_standard_deviation (Laplace/Gaussian mechanisms) and (epsilon, delta) (generic mechanism). There are multiple way how to do this. Here it is assumed that (epsilon, delta) specifies Laplace mechanism and epsilon is computed based on this. The delta is computed to be proportional to epsilon.

Reply:
Got it 👍 Please let me know if my edits do not make sense. Thanks!